README.md

```diff
@@ -9,6 +9,91 @@ It can be used in two modes:
 a legacy mode in which there is a single Git repository hosted by Jenkins itself, to which you may push changes;
 and a more general mode in which you may define libraries hosted by any SCM in a location of your choice.
 
+Older versions of this plugin may not be safe to use. Please review the
+following warnings before using an older version:
 
-Comprehensive user documentation can be found [in the Pipeline chapter of the
-User Handbook](https://jenkins.io/doc/book/pipeline/shared-libraries/).
+-   [Missing permission
+    check](https://jenkins.io/security/advisory/2019-07-31/#SECURITY-1422)
```

Comment:
The Older version warning should be removed (the plugin page will add it automatically)